runs/graphql/server.js

```diff
@@ -1,17 +1,51 @@
 const express = require("express");
 const { ApolloServer, gql } = require("apollo-server-express");
+const data = require("../../_posts/_data.json");
+const { readFileSync } = require("fs");
+const fm = require("front-matter");
+const marked = require("marked");
 
 // Construct a schema, using GraphQL schema language
 const typeDefs = gql`
+  type Event {
+    title: String
+    markdown: String
+    content: String
+    link: String
+    date: String
+  }
   type Query {
     hello: String
+    events: [Event]
   }
 `;
 
 // Provide resolver functions for your schema fields
 const resolvers = {
   Query: {
-    hello: () => "Hello world!"
+    hello: () => "Hello world!",
+    events: () => {
+      return data.posts.map(post => {
+        const markdown = readFileSync("../../_posts/" + post, "utf8");
+        const parsed = fm(markdown);
+        const date = parsed.attributes.date
+          ? new Date(parsed.attributes.date + " GMT+0200")
```

Comment:
This might lead to unexpected errors if the date is entered wrongly, I suppose. Parsing with a date parsing library like moment or luxon would enable us to catch errors a bit better.

Reply:
I like dayjs or luxon, that could be good to parse different formats.
But I don't think luxon can magically fix wrongly entered data? 😅